swan-lake/learn-the-language/distinctive-language-features/data.md

```diff
@@ -109,7 +109,7 @@ Ballerina also has a concept of a configurable variable. A module-level variable
 configurable int port = 8080;
 ```
 
-The initializer of a configurable variable can be overridden at runtime. A variable where runtime configuration is required can use ``?`` as the initializer.
+The initializer of a configurable variable can be overridden at runtime. A variable where runtime configuration is required can use ``?`` as the initializer. Values for these variables can be provided via `Config.toml` file.
```

Comment:
I wonder if this could be misleading since there's other ways to provide values.

Reply:
The only way is to provide it via the `Config.toml` right?

Reply:
See https://ballerina.io/learn/configure-ballerina-programs/provide-values-to-configurable-variables/.

Reply:
Updated in [43f8e07](https://github.com/ballerina-platform/ballerina-dev-website/pull/5711/commits/43f8e076b4456c2cb75023dc01630e0d983a03e6)